src/main/java/games/strategy/persistence/memento/serializable/SerializableMementoImporter.java

```diff
@@ -0,0 +1,45 @@
+package games.strategy.persistence.memento.serializable;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.ObjectInputStream;
+
+import org.apache.commons.io.input.CloseShieldInputStream;
+
+import games.strategy.util.memento.Memento;
+
+/**
+ * A memento importer for the Java object serialization format.
+ *
+ * <p>
+ * Instances of this class are immutable.
+ * </p>
+ */
+public final class SerializableMementoImporter {
+  /**
+   * Imports a memento from the specified stream.
+   *
+   * @param is The stream from which the memento will be imported. The caller is responsible for closing this stream; it
+   *        will not be closed when this method returns.
+   *
+   * @return The imported memento.
+   *
+   * @throws SerializableMementoImportException If an error occurs while importing the memento.
+   */
+  @SuppressWarnings("static-method")
```

Comment:
@RoiEXLab I have that warning turned on in my Eclipse workspace, so I probably just Quick Fixed it without realizing it.  Since not everyone has their static analysis tools configured the same way, I'll remove it in a PR I plan on submitting tomorrow to address the rest of @DanVanAtta's comments from this PR.  Thanks for the heads up!